try:
    num=int(input())
    print(num)
except Exception as e:
    pass